excercsises/3_synchronization/contact.spec.js

```diff
@@ -11,8 +11,6 @@ describe("Protractor Workshop app", function () {
 
   it('should display text "Your message has been sent." when user sends message  ', function () {
     const success = "Your message has been sent.";
-    contactPage.fillForm("name name", "name.noname@gmail.com", "content content");
-    contactPage.submitButton();
-    expect(contactPage.checkSucces(success)).toEqual(success);
+    expect(contactPage.fillFormSubmitCheckSuccess("name name", "name.noname@gmail.com", "content content","Your message has been sent.")).toEqual(success);
```

Comment:
In general it's better now, but would be perfect if you can split this for two user action:
1) fillFormAndSubmit(...)
2) getSuccessInformation()

As we spoke during the day - it's fine to group the actions per one big action, but here you took everything and put it into one method. We can speak more about it on tomorrow call.